app/src/main/java/fr/free/nrw/commons/category/CategoryImagesListFragment.java

```diff
@@ -143,10 +143,17 @@ private void handleNoInternet() {
      * @param throwable
      */
     private void handleError(Throwable throwable) {
-        Timber.e(throwable, "Error occurred while loading images inside a category");
         try{
-            ViewUtil.showShortSnackbar(parentLayout, R.string.error_loading_images);
-            initErrorView();
+            if (throwable.getMessage().contains("The mapper function returned a null value")){
```

Comment:
@chinmoy12c  Why are we using error strings for showing the message, the logs clearly show that there is a null-pointer exception, my guess is that responseToMediaList in Media client is receiving a null parameter and it is there where we need to make changes to show the appropriate message